src/main/java/club/sk1er/patcher/config/PatcherConfig.java

```diff
@@ -141,11 +141,33 @@
 
     @Property(
         type = PropertyType.SWITCH, name = "Disable Titles",
-        description = "Stop titles & subtitles from appearing.",
-        category = "Miscellaneous", subcategory = "Overlays"
+        description = "Stop titles from appearing.",
+        category = "Miscellaneous", subcategory = "Titles"
```

Comment:
This will break people's config files. Please keep the existing subcategory, and change `i18nSubcategory` instead.

Reply:
Done